js/lykke2.js

```diff
@@ -558,11 +560,12 @@ module.exports = class lykke2 extends Exchange {
         //         Price: 9847.427,
         //         Fee: { Amount: null, Type: 'Unknown', FeeAssetId: null }
         //     },
+        //
         const marketId = this.safeString (trade, 'assetPairId');
         const symbol = this.safeSymbol (marketId, market);
         const id = this.safeString2 (trade, 'id', 'id');
         const orderId = this.safeString (trade, 'orderId');
-        const timestamp = this.parse8601 (this.safeString2 (trade, 'timestamp', 'timestamp'));
+        const timestamp = this.safeNumber (trade, 'timestamp', 'timestamp');
```

Comment:
the timestamp should be an integer here, and the second argument is incorrect, should just be:

```
this.safeInteger (trade, 'timestamp')
```